airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,1609 @@
+version: 0.71.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - projects
+streams:
+  - type: DeclarativeStream
+    name: projects
+    primary_key:
+      - id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $schema: http://json-schema.org/schema#
+        properties:
+          account_id:
+            type: number
+          atom_enabled:
+            type: boolean
+          automatic_planning:
+            type: boolean
+          bugs_and_chores_are_estimatable:
+            type: boolean
+          created_at:
+            type: string
+          current_iteration_number:
+            type: number
+          enable_following:
+            type: boolean
+          enable_incoming_emails:
+            type: boolean
+          enable_tasks:
+            type: boolean
+          has_google_domain:
+            type: boolean
+          id:
+            type: number
+          initial_velocity:
+            type: number
+          iteration_length:
+            type: number
+          kind:
+            type: string
+          name:
+            type: string
+          number_of_done_iterations_to_show:
+            type: number
+          point_scale:
+            type: string
+          point_scale_is_custom:
+            type: boolean
+          profile_content:
+            type: string
+          project_type:
+            type: string
+          public:
+            type: boolean
+          show_priority_icon:
+            type: boolean
+          show_priority_icon_in_all_panels:
+            type: boolean
+          show_story_priority:
+            type: boolean
+          start_time:
+            type: string
+          time_zone:
+            properties:
+              kind:
+                type: string
+              offset:
+                type: string
+              olson_name:
+                type: string
+            type: object
+          updated_at:
+            type: string
+          velocity_averaged_over:
+            type: number
+          version:
+            type: number
+          week_start_day:
+            type: string
+        type: object
+    retriever:
```

Comment:
The `retriever` parameter seems to be the same for all streams - could you please declare it separately and re-use?